src/plugins/label/styles.scss

```diff
@@ -0,0 +1,29 @@
+.ExpandableTestWidget {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    flex-grow: 1;
+    align-self: stretch;
+    // background: var(--accent);
+    cursor: pointer;
```

Comment:
Label isn't actually clickable so I think it's better to remove this line and keep default cursor